source/platform/sighandl.cpp

```diff
@@ -6,6 +6,7 @@
 #include <internal/sighandl.h>
 #include <unistd.h>
 #include <fcntl.h>
+#include <sys/param.h>
```

Comment:
It compiles fine for me without this header.

Reply:
This could be because you used gcc instead of clang. My system does not have gcc installed

Reply:
Okay. I will try with clang instead.